README.md

```diff
@@ -138,6 +137,44 @@ Below is a typical example of how this use case would take place:
 
 - Compute the new root for the vault’s rewards distributor, submit it, wait for the timelock (if any), accept the root, and let vault depositors claim their rewards according to the vault manager’s rewards re-distribution strategy.
 
+## Emergency
+
+### An enabled market risk is now considered unsafe
+
+If the owner/curator consider an enabled market unsafe (e.g., risk too high) and want to disable this market, he should follow these steps :
+
+- Set the cap of the market to 0 with the `submitCap` function.
+- Revoke the pending cap of the market with the `revokePendingCap` function (this can also be done by the guardian).
+- Withdraw all the supply of this market with the `reallocate` function. If there is not enough liquidity on the market, the market should be put at the beginning of the withdraw queue (with the `updateWithdrawQueue` function).
```

Comment:
```suggestion
- Withdraw all the supply of this market with the `reallocate` function.
  If there is not enough liquidity on the market, the market should be put at the beginning of the withdraw queue (with the `updateWithdrawQueue` function).
```
putting each sentence on its own line is recommended so that changes are easier to track with git